src/test/java/bio/terra/app/controller/SnapshotsApiControllerTest.java

```diff
@@ -73,25 +86,32 @@ class SnapshotsApiControllerTest {
 
   private static final AuthenticatedUserRequest TEST_USER =
       AuthenticationFixtures.randomUserRequest();
+  private static final UUID DATASET_ID = UUID.randomUUID();
   private static final UUID SNAPSHOT_ID = UUID.randomUUID();
   private static final String JOB_ID = "a-job-id";
+  private static final JobModel JOB_MODEL =
+      new JobModel().id(JOB_ID).jobStatus(JobModel.JobStatusEnum.RUNNING);
   private static final Boolean EXPORT_GCS_PATHS = false;
   private static final Boolean VALIDATE_PRIMARY_KEY_UNIQUENESS = true;
   private static final Boolean SIGN_URLS = true;
   private static final String TABLE_NAME = "table1";
   private static final String COLUMN_NAME = PDAO_ROW_ID_COLUMN;
   private static final int LIMIT = 100;
   private static final int OFFSET = 0;
+  private static final EnumerateSortByParam SORT = EnumerateSortByParam.NAME;
   private static final SqlSortDirectionAscDefault DIRECTION = SqlSortDirectionAscDefault.ASC;
   private static final String FILTER = "";
+  private static final String REGION = "a-region";
+  private static final String TAG = "a-tag";
+  private static final String DUOS_ID = "a-duos-id";
+  private static final SnapshotRetrieveIncludeModel INCLUDE = SnapshotRetrieveIncludeModel.TABLES;
 
-  private static final String RETRIEVE_SNAPSHOT_ENDPOINT = "/api/repository/v1/snapshots/{id}";
-  private static final String LOCK_SNAPSHOT_ENDPOINT = "/api/repository/v1/snapshots/{id}/lock";
-  private static final String UNLOCK_SNAPSHOT_ENDPOINT = "/api/repository/v1/snapshots/{id}/unlock";
-
-  private static final String QUERY_SNAPSHOT_DATA_ENDPOINT =
-      RETRIEVE_SNAPSHOT_ENDPOINT + "/data/{table}";
-  private static final String EXPORT_SNAPSHOT_ENDPOINT = RETRIEVE_SNAPSHOT_ENDPOINT + "/export";
+  private static final String SNAPSHOTS_ENDPOINT = "/api/repository/v1/snapshots";
+  private static final String SNAPSHOT_ENDPOINT = SNAPSHOTS_ENDPOINT + "/{id}";
```

Comment:
```suggestion
  private static final String SNAPSHOT_ENDPOINT_ID = SNAPSHOTS_ENDPOINT + "/{id}";
```

nit: consider differentiating this endpoint more from `SNAPSHOTS_ENDPOINT`

Reply:
I'll change to `SNAPSHOT_ID_ENDPOINT`, because ending with `_ID` suffix leads me to think that the constant itself is an ID. Thanks!